README.md

```diff
@@ -85,16 +87,14 @@ Earlier in 2023 we started to treemend an existing `monaco-editor` dependency vi
 
 #### Overrides instructions
 
-With v7 we decided to use readily treemended version of monaco-editor called [@codingame/monaco-editor-treemended](https://www.npmjs.com/package/@codingame/monaco-editor-treemended), but this requires to add `overrides` (npm/pnpm) and `resolutions` (yarn) in your project. Setting these ensures that all dependencies to `monaco-editor` and `vscode` are aligned:
+With v7 we decided to use readily treemended version of monaco-editor called [@codingame/monaco-vscode-editor-api](https://www.npmjs.com/package/@codingame/monaco-vscode-editor-api), but this requires to add `overrides` (npm/pnpm) and `resolutions` (yarn) in your project. Setting these ensures that all dependencies to `monaco-editor` and `vscode` are aligned:
 
 ```yaml
   "overrides": {
-    "monaco-editor": "npm:@codingame/monaco-editor-treemended@>=1.85.5 <1.86.0",
-    "vscode": "npm:@codingame/monaco-vscode-api@>=1.85.5 <1.86.0"
+    "monaco-editor": "npm:@codingame/monaco-vscode-editor-api@>=2.0.0 <2.1.0"
   },
   "resolutions": {
-    "monaco-editor": "npm:@codingame/monaco-editor-treemended@>=1.85.5 <1.86.0",
-    "vscode": "npm:@codingame/monaco-vscode-api@>=1.85.5 <1.86.0"
+    "monaco-editor": "npm:@codingame/monaco-vscode-editor-api@>=2.0.0 <2.1.0"
```

Comment:
why removing the vscode override?

Reply:
I came to the conclusion it is not needed. The dependency is declared and there usually is no mismatching package.